### PR TITLE
@ljwolf final pass on clustering

### PR DESCRIPTION
**Pull request by ljwolf**
This is my final pass on clustering. I think @darribas already had one go, but I've re-written a bit of the exposition to use scaled covariates, rather than raw covariates, in the clustering. 

This only changes a few of the interpretation paragraphs, and (of course) changes the cluster visualisations. 

I aim to get this done fully tomorrow.

### CODE REVIEW

File: notebooks/10_clustering_and_regionalization.md [hunk 1]
```diff
@@ -802,23 +840,81 @@ connectivity graph modeled by our weights matrix. Therefore, using k-nearest nei
...
+```
+
+For all functions in `metrics` that end in "score", higher numbers indicate greater fit, whereas functions that end in `loss` work in the other direction. Thus, the K-means solution has the highest Calinski-Harabasz score, while the ward clustering comes second. The regionalizations both come *well* below the clusterings, too. As we said before, the improved geographical coherence comes at a pretty hefty cost in terms of feature goodness of fit. This is because regionalization is *constrained*, and mathematically *can not* achieve the same score as the unconstrained K-means solution, unless we get lucky and the k-means solution *is* a valid regionalization. 
+
+
+### Solution Similarity
+
+The `metrics` module also contains useful tools to compare whether the labellings generated from different clustering algorithms are similar, such as the Adjusted Rand Score or the Mutual Information Score. To show that, we can see how similar clusterings are to one another:
+
+```python
```
Comment (darribas) [inline notebooks/10_clustering_and_regionalization.md:899]: I'd document each line in this cell as it's a bit complex

**Review by darribas — CHANGES_REQUESTED** [review]
Excellent edits! Only minor comments fro my end

File: notebooks/10_clustering_and_regionalization.md [hunk 2]
```diff
@@ -242,11 +237,12 @@ So, the fact that all of the clustering variables are positively autocorrelated
 say much about how attributes co-vary over space. To explore cross-attribute relationships,
 we need to consider the spatial correlation between variables. We will take our first dip
 in this direction exploring the bivariate correlation in the maps of covariates themselves.
-This would mean that we would be comparing each pair of chroplets to look for associations
+This would mean that we would be comparing each pair of chropleths to look for associations
```
[the remark this answers is not in the document]
Comment (ljwolf) [reply notebooks/10_clustering_and_regionalization.md:240]: Hmm, I thought I caught this? will add.

File: notebooks/10_clustering_and_regionalization.md [hunk 3]
```diff
@@ -434,18 +468,17 @@ And, to show this visually:
 _ = areas.plot.bar()
 ```
 
-Our visual impression is confirmed: cluster 3 contains tracts that
-together comprise 6,636 square kilometers (approximately 2562 square miles),
-which accounts for over half of the total land area in the county:
+Our visual impression is confirmed: cluster 1 contains tracts that
+together comprise 6,636 square kilometers (nearly 8000 square miles),
```
[the remark this answers is not in the document]
Comment (ljwolf) [reply notebooks/10_clustering_and_regionalization.md:472]: OK, I think both are wrong,. The aggregation occurs over square miles, so the figure in the above cell shows the region area is 8,622 square miles. That'd be about 22330 square kilometres. I've fixed the text to resolve this.

**Comment by ljwolf** [comment]
Great, OK, I've addressed the points raised.

**Pull request merged**